rfq/oracle_test.go

```diff
@@ -95,7 +117,7 @@ func runQueryAskPriceTest(t *testing.T, tc *testCaseQueryAskPrice) {
 	defer backendService.Stop()
 
 	// Wait for the server to start.
-	time.Sleep(2 * time.Second)
+	time.Sleep(200 * time.Millisecond)
```

Comment:
Is it intended to reduce the time, rather than be equivalent?

Reply:
The whole test took 14 seconds because of this wait, which is unnecessary (it's an in-memory server which should be spun up in a couple of milliseconds, even on slow systems).